cumulus_etl/etl/studies/covid_symptom/covid_tasks.py

```diff
@@ -118,11 +122,25 @@ async def read_entries(self, *, progress: rich.progress.Progress = None) -> task
                 self.add_error(orig_docref)
                 continue
 
+            # Record this docref as processed (so we can delete it from our table if we don't generate symptoms).
+            #
+            # We could move this up to the beginning of the loop and delete symptoms for notes now marked as
+            # superseded or are no longer considered an ED docref, for database cleanliness.
+            # But the current approach instead focuses purely on accuracy and makes sure that we zero-out any dangling
+            # entries for groups that we do process.
+            # Downstream SQL can ignore the above cases itself, as needed.
```

Comment:
⚠️⚠️  - do you have a run someplace where all the covid NLP data is zeroed out? can we make sure the library still builds successfully?

Reply:
1) No I don't right now
2) I wouldn't worry about the entire database going to zero - that would be a true oddity. This PR is more worried about a single document's symptoms going to zero.
3) My comments about downstream SQL there are more like "if document A suddenly gets marked as entered-in-error, let's not bother deleting their symptoms in Athena -- the study should be ignoring entered-in-error docs anyway"

I would expect no changes in the likelihood of the library building or not - this is more like, a couple symptoms might disappear from the database.

Reply:
got it - makes sense